### PR TITLE
Introduction 8

### PR DESCRIPTION
**Pull request by icedandreas**
Fixes: #96 
Done to my liking.
It has a lot of rather open questions that gets hanled by the 2 problem statements later on, but its not to everyones liking, so need opinions on it.
Also removed Neurospace from the introduction as I do not feel it belongs there.

### CODE REVIEW

File: report/Introduction/IntroductionIntro.tex [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Since the beginning of the industrial revolution in the late 1700s, industrial equipment has needed maintenance.
+The maintenance would often be to apply grease to parts under large friction from moving parts and oil in metal to prevent rust. 
+Since, the way that maintenance is performed has changed, not only in complexity but also in maintenance strategies. \cite{MaintenanceScheduling}
+
+While maintenance strategies have changed since the industrial revolution, the importance and widespread use of maintenance have not.
+A recent study shows that "34\% of facilities are spending over 30 hours/week on scheduled maintenance" and that "... the average time spent on scheduled maintenance is 19 hours/week." \cite{MaintenanceHours}. 
+The impact of maintenance is furthermore proven by the fact that 40 to 50\% of operational budgets in the industry are spent on maintenance \citep{MaintenanceBudget}.
+
+There are many different types of maintenance programmes that are or have been, used throughout history.
```
Comment (thomaslhansen) [inline report/Introduction/IntroductionIntro.tex]:
```suggestion
There are many different types of maintenance strategies that are or have been, used throughout history.
```

File: report/Introduction/IntroductionIntro.tex [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+The many types of maintenance programmes are divided into two categories: proactive and reactive.
```
Comment (thomaslhansen) [inline report/Introduction/IntroductionIntro.tex]:
```suggestion
The many types of maintenance strategies are divided into two categories: proactive and reactive.
```

File: report/Introduction/IntroductionIntro.tex [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+The main difference between the two categories is that proactive maintenance tries to deal with problems before they occur while reactive maintenance focuses on dealing with the problems after they occur.
+The subcategories within both proactive and reactive maintenance each deploy their own set of methods to perform maintenance.
+A study from 2019 concerning maintenance found that 56\% of manufacturing facilities use reactive maintenance, 78\% use preventive maintenance and 43\% use predictive maintenance \cite{2019MaintenanceStudy}.
+
+The scope of this project is limited to predictive maintenance since it is an interesting field that is also relevant to the field of software engineering.
+We will still briefly touch on the topics of reactive maintenance and proactive maintenance in \autoref{cha:maintenance}.
+
+When monitoring machines, there is a huge array of possible data to collect, however, it is possible that not all the data will be useable for performing predictive maintenance.
+Furthermore, as mentioned earlier, predictive maintenance is the least used form of maintenance.
+What is the reason behind this?
```
Comment (thomaslhansen) [inline report/Introduction/IntroductionIntro.tex]: Jeg syntes måske den slutter lidt underligt med tre spørgsmål og aldrig rigtigt nogen slutning.

Comment (icedandreas) [reply report/Introduction/IntroductionIntro.tex]: Har tilføjet en lille outro der leder den røde tråd videre til initial problem statement.

**Pull request closed without merging**